common/utils.js

```diff
@@ -1283,6 +1282,20 @@
                     }
                 }
             }
+
+            if (chaiseConfig.configRules) {
```

Comment:
I feel like we should do more checking here. If any of these properties are ill-defined, this implementation will throw a terminal error. These are the changes that I suggest:

- change line 1286 check to `Array.isArray(chaiseConfig.configRules)`
- Based on the issue, `configRules` is written as `config-rules`. Either change the implementation or documentation. 
- Based on the issue, `host` can be a single string too. So you could just check for that first and then turn it into an array of one.
- Make sure `host` is array and then do the `forEach` (`Array.isArray(ruleset.host`). 
- (not important) since you just need to match with one of the hosts in the ruleset.host, you can turn the `ruleset.host.forEach` into a simple loop and break out of it when the host matches. Although since the host array is usually small, this shouldn't matter.
- Make sure `ruleset.config` is defined and is an object before looping through its properties.